client/blocks/product-purchase-features-list/custom-css.jsx

```diff
@@ -38,7 +38,7 @@ export default localize( ( { selectedSite, translate } ) => {
 				icon={ <img alt="" src={ customizeImage } /> }
 				title={ translate( 'Custom CSS' ) }
 				description={ translate(
-					"Make advanced changes to your site's appearance by writing your own CSS."
+					"Enjoy more control over your site's look and feel by writing your own CSS."
```

Comment:
Not new here, but I think we prefer "fancy" punctuation when posible. In this case: `’` over `'`.
```suggestion
					'Enjoy more control over your site’s look and feel by writing your own CSS.'
```